cmake/Modules/FindReadline.cmake

```diff
@@ -0,0 +1,53 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
I've never written a cmake file before, this was created by slowly removing parts of FindOpenCL.cmake, I'd appreciate if someone familiar with cmake could take a harder look at this.